components/private_channel/browser/private_channel.cc

```diff
@@ -0,0 +1,298 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "brave/components/private_channel/browser/private_channel.h"
+
+#include <utility>
+
+#include "brave/components/private_channel/browser/constants.h"
+#include "brave/components/private_channel/browser/request_utils.h"
+#include "brave/components/private_channel/client_private_channel.h"
+#include "brave/components/private_channel/utils.h"
+
+#include "base/logging.h"
+#include "base/task/post_task.h"
+#include "chrome/browser/browser_process.h"
```

Comment:
chrome/browser deps are not allowed in components. See "Recipes for Breaking //chrome Dependencies" https://www.chromium.org/developers/design-documents/cookbook